### DISCUSSION TITLE
made my plot clear

### DISCUSSION THREAD

**Discussion by HamzaAli195** [AammarTufail/python_ka_chilla2023#59] [Q&A] [answered]
the values on x-axis  are merged with each other how i can resolve them?
following is the pic.
<img width="527" alt="rec" src="https://user-images.githubusercontent.com/121845773/212164280-494706f4-1a8d-4c4b-a787-398afb1c0d4d.PNG">

**Comment by usamabajwa86** [comment, marked as answer]
export your charts as .BMP or .TIFF format and check in windows image viewer or show axis as vertical
using command

plt.xticks(rotation=90)
